### PR TITLE
issue-30g

### PR DESCRIPTION
**Pull request by mishaone**
* Reorganized a bunch of modules
* Added analyze content
* Added advanced che exploration content

### CODE REVIEW

**Comment by mishaone** [comment]
@rkratky - requesting a review because I came into some issues with the tip not being in the expected place, etc when pushing these final commits. Could you do a quick sanity check to make sure nothing seems too broken and merge please? I'll do the final part in a new PR tomorrow.

File: docs/topics/modules/e2e/about_oso_projects.adoc [hunk 1]
```diff
@@ -1,10 +1,14 @@
 [#about_oso_projects]
-= About Projects in OpenShift Online
+= Viewing Projects in OpenShift Online
 
-To view the OpenShift Online projects that support your project's pipeline, navigate to https://console.starter-us-east-2.openshift.com/. This page displays the following projects/namespaces that are created in OpenShift Online:
+To view the OpenShift Online projects that support your project's pipeline, navigate to https://console.starter-us-east-2.openshift.com/.
+
+image::oso_projects.png[OpenShift Online Projects]
+
+This page displays the following projects/namespaces that are created in OpenShift Online:
```
Comment (rkratky) [inline docs/topics/modules/e2e/about_oso_projects.adoc]: Slashes should be avoided in sentences. Can we use 'and' instead?

File: docs/topics/modules/e2e/analyze_che_dependencies.adoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+[source,xml]
+----
+<dependency>
+  <groupId>ch.qos.logback</groupId>
+  <artifactId>logback-core</artifactId>
+  <version>1.1.0</version>
+</dependency>
+----
++
+. An error icon appears near the line numbers when you add the XML code. Mouse over the icon to view the error message (see screen capture).
```
Comment (rkratky) [inline docs/topics/modules/e2e/analyze_che_dependencies.adoc]: 'Mouse over' as a verb is too jargony. The IBM Style Guide suggests 'point to' or 'move the mouse pointer over'.

File: docs/topics/modules/e2e/analyze_che_dependencies.adoc [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
++
+image::red_x.png[Error Icon]
++
+The dependency analyzer displays this error to indicate that the current version (1.1.0) has a CVE (Common Vulnerabilities and Exposures). It also recommends using version 1.2.1 instead to fix the problem.
```
Comment (rkratky) [inline docs/topics/modules/e2e/analyze_che_dependencies.adoc]: "has a CVE" would probably be better to expand a little, such as "has a CVE issue associated with it".

File: docs/topics/modules/e2e/analyze_che_dependencies.adoc [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+. Update the version in the *version* tags of the code to `1.2.1`. The red icon disappears when the version number is updated.
```
Comment (rkratky) [inline docs/topics/modules/e2e/analyze_che_dependencies.adoc]: Perhaps use "`<version>` tags", i.e. incl. the brackets, for clarity?

File: docs/topics/modules/e2e/cleanup_sb.adoc [hunk 5]
```diff
@@ -1,23 +1,27 @@
...
++
+. In the dialog box, add your {osio} user name and click btn:[I understand my actions - erase my environment].
++
+image::are_you_sure.png[Confirm]
++
+. After the environment is successfully erased, the success page displays:
++
+image::success.png[Success]
++
+. Click the check mark icon in the top right side of the page to see the `Jenkins` and `Che` status. Allow several minutes for this to load. When all the status indicators are green (see screen capture), you can use your {osio} account again. If the status indicators are not green after several minutes, see <<che_jenkins_status>> for troubleshooting information.
```
Comment (rkratky) [inline docs/topics/modules/e2e/cleanup_sb.adoc]: `s/in the top right side/on the top right/`

File: docs/topics/modules/e2e/create_springboot.adoc [hunk 6]
```diff
@@ -1,15 +1,23 @@
 [#create_springboot]
 = Creating a Spring Boot Quickstart Project
 
+You will not learn how to create a second quickstart project in your Space. The Hello World project was a Vert.X application and the second quickstart is a Spring Boot application.
```
Comment (rkratky) [inline docs/topics/modules/e2e/create_springboot.adoc]: - `s/You will not/You will now/`?
- `s/application and/application, and/`

File: docs/topics/modules/e2e/create_springboot.adoc [hunk 7]
```diff
@@ -1,15 +1,23 @@
...
 . In your space's dashboard, click btn:[Add to space].
-. Select `Create a new Quickstart project`.
+. Click btn:[Create a new Quickstart project].
++
+image::get_started_menu.png[Getting Started Menu]
++
 . In the `Application Wizard`:
-.. Click a project type from the displayed icons. For this example, click btn:[Spring Boot - HTTP].
+.. In the `Name` field, type *spring* and press kbd:[Enter] to filter the options for Spring Boot quickstart applications.
+.. From the displayed results, click btn:[Spring Boot - HTTP].
```
Comment (rkratky) [inline docs/topics/modules/e2e/create_springboot.adoc]: I don't think this thing qualifies as a 'button'. Maybe call it 'item' or just 'option'?

File: docs/topics/modules/e2e/reduce_mem_sb.adoc [hunk 8]
```diff
@@ -1,16 +1,20 @@
 [#reduce_mem_sb]
 = Reduce the Boot Spring Project's Memory Usage
 
-Your {osio} space now has two quickstart projects running: the Hello World Vert.x project and the new Spring Boot HTTP project. These two projects share the same limited memory resources in OpenShift Online. You must therefore reduce the memory usage for these projects.
+Your OpenShift Online account must now share resources with two quickstart projects: the Hello World Vert.x project and the new Spring Boot HTTP project. 
 
 See <<reduce_memory>> to optimize memory usage for the Hello World project and use the following instructions for the Spring Boot project:
```
Comment (rkratky) [inline docs/topics/modules/e2e/reduce_mem_sb.adoc]: `s/project and use/project, and use/`

File: docs/topics/modules/e2e/using_code_assistant.adoc [hunk 9]
```diff
@@ -0,0 +1,35 @@
+[#using_code_assistant]
+= Using the Code Assistant
+
+. Return to the Che workspace browser tab. If you closed your earlier workspace tab, use the following instructions:
+.. In {osio}, click the btn:[Create] tab.
+.. The default `Codebases` view lists your project workspace:
++
+image::existing_workspace.png[Existing Workspace]
++
+.. Click btn:[Open] to view the existing workspace in a new tab. If the new tab does not load, ensure your browser is not blocking pop ups.
```
Comment (rkratky) [inline docs/topics/modules/e2e/using_code_assistant.adoc]: As per the IBM Style Guide: `s/pop ups/pop-up windows/`.

File: docs/topics/modules/e2e/using_code_assistant.adoc [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+.. When the workspace loads, navigate to the following file: `src` > `main` > `Java` > `io.openshift.booster` > `HttpApplication.java`.
++
+image::project_dir.png[Project Directory]
+. In the `HttpApplication.java` file, add a line break at line 9 after the imports section.
+. Type the following in the new line:
++
+```
```
Comment (rkratky) [inline docs/topics/modules/e2e/using_code_assistant.adoc]: Replace ``` with

```
[source,java]
----
foo
----
```

File: docs/topics/modules/e2e/using_code_assistant.adoc [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+import io.
+```
++
+. Press kbd:[Ctrl+Space] to view the options for this line and select any one of the options.
++
+image::code_options.png[Code Autocomplete Options]
++
+. Mouse over the error icon next to the line number to view the error, which warns you that the import is not used.
```
Comment (rkratky) [inline docs/topics/modules/e2e/using_code_assistant.adoc]: 'mouse over' -- see above.

File: docs/topics/modules/e2e/using_stack_reports.adoc [hunk 12]
```diff
@@ -0,0 +1,27 @@
+[#using_stack_reports]
+= Using Stack Reports
+
+{osio} includes a Stack Report feature that provides information about your stack components and recommends improvements for your application. The Stack Report is available after the first the first pipeline build completes.
```
Comment (rkratky) [inline docs/topics/modules/e2e/using_stack_reports.adoc]: Repeated words: "the first the first".

File: docs/topics/modules/e2e/using_terminal_tab.adoc [hunk 13]
```diff
@@ -0,0 +1,45 @@
...
++
+image::tri_run.png[Run Icon]
++
+. Click *Terminal* for the terminal tab at the bottom of your workspace:
++
+image::terminal_start.png[Terminal Tab]
++
+. Type and execute the following command in the terminal tab:
++
+```
```
Comment (rkratky) [inline docs/topics/modules/e2e/using_terminal_tab.adoc]: Four dashes in place of three backticks.

In this whole file...

**Comment by rkratky** [comment]
@mishaone, forgot to add the I verified that the PR builds fine, so it's OK to merge when the comments are addressed.

**Comment by mishaone** [comment]
@rkratky I have no idea how to pull in changes to my local branch that someone else made to this PR. Any  thoughts? I suspect going in blind will just cause more conflicts as they did when I incorporated your merged issue 61 changes earlier.

**Comment by mishaone** [comment]
Fixed all the things identified, with a few deviations from the comma suggestions. Local build was okay and validation script was fine so merging this so we can start a new PR tomorrow.

**Pull request merged**